src/components/calculator.css

```diff
@@ -0,0 +1,84 @@
+.calculator {
+  /* Size and Position */
+  height: 80vh;
+  width: 50vh;
+  min-width: 300px;
+  min-height: 500px;
+  padding: 10px;
+
+  /* Layout */
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+
+  /* Design */
+  background: rgba(255, 255, 255, 0.25);
+  border: 3px solid white;
+  border-radius: 15px;
+  box-shadow: 0 0 20px black;
```

Comment:
- [ ] You made great styles for your calculator. However, you should follow the requirements.

![Styles-calculator](https://user-images.githubusercontent.com/79876959/144699979-25b92e5f-3da8-4e2b-a46d-b0d295304706.PNG)

**It means to use the same colors, text, and size for the buttons.**

Reply:
Fixed it. Adjusted as per the design